examples/front-proxy/docker-compose.yaml

```diff
@@ -18,13 +17,17 @@ services:
   service1:
     build:
       context: ../shared/python
-      target: aiohttp-tracing-service2
+      target: aiohttp-tracing-service3
+    volumes:
+    - ./service-envoy.yaml:/etc/service-envoy.yaml
```

Comment:
thanks for follow up @eitansuez 

im thinking we need to add separate envoy services for these sidecars

Reply:
both sidecar envoys listen on port 8000 and route to their local service listening on port 8080.
in what way would two distinct service-envoy.yaml's differ?

Reply:
i didnt mean that the sidecars require different config to each other

i meant that this is injecting the config into the backend service

Reply:
you can just update this PR - no need to create new one

Reply:
ok, got it. thanks for the clarification.